db/seeds.rb

```diff
@@ -369,6 +369,8 @@
                       "Consider <a href='/settings'>connecting it</a>.",
     github_connect: "You're on a roll! 🎉  Do you have a GitHub account? " \
                     "Consider <a href='/settings'>connecting it</a> so you can pin any of your repos to your profile.",
+    google_oauth2_connect: "You're on a roll! 🎉  Do you have a Google account? " \
```

Comment:
Seems like we're using this in 3 places (here, the DUS and the specs),  maybe a constant could make sense?

Reply:
I agree, but I want to do this right and constantize all the messages across all the OAuth providers. I'm going to punt it to a separate PR after this one.